### ISSUE TITLE
[Quest] The Smallest Creatures

### ISSUE THREAD

**Issue by netherwingbta** [Atlantiss/NetherwingBugtracker#2246]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)

**Description**: 
https://www.wowhead.com/quest=10720/the-smallest-creatures
Enter a shadow realm where you cannot see enemy mobs.
You are also unable to poison the kegs, says "Ability is not ready yet" when you try to use the poison keg spell or
"You are too far away" if you right click on it an
**Current behaviour**:
![2018-11-04-0216-18 mp4_snapshot_00 51_ 2018 11 04_02 19 12](https://user-images.githubusercontent.com/43872263/47959011-fe77e380-dfd8-11e8-8bce-aa698b6c7a03.jpg)
![2018-11-04-0222-18 mp4_snapshot_00 53_ 2018 11 04_02 31 35](https://user-images.githubusercontent.com/43872263/47959035-d8067800-dfd9-11e8-9fcd-231fc56ff941.jpg)
Uncompletable atm. 
Stealth Ability also is missing.

**Expected behaviour**: 
![2018-11-04-0222-18 mp4_snapshot_00 53_ 2018 11 04_02 31 35](https://user-images.githubusercontent.com/43872263/47959050-39c6e200-dfda-11e8-924c-99027bf6b1cc.jpg)
https://www.youtube.com/watch?v=Amsn4iBbxAI&t=15s
https://www.youtube.com/watch?v=Azmzdz8D510&t=60

**Server Revision**: 
2244

**Comment by aresbrutus** [comment]
https://github.com/Atlantiss/AtlantissTBCBT/issues/741

**Comment by mhdemn** [comment]
2282

Wanna update with some video footage:


> Enter a shadow realm where you cannot see enemy mobs.

No longer shadowy, but still can't see enemy mobs as seen [here](https://youtu.be/99W3RRNWqzo). 

Marmot pulls invisible ogres, they kill it, immediately turn to the player. The buff seems to phase you to this NPC-blinding realm and give you a shoddy stealth.


> Stealth Ability also is missing.

Stealth is in the marmot's bar now. However, [here](https://youtu.be/_aoFfXl_Tm8) you can see that when you use the spell the marmot will get fixed in place until she's dismissed.


> You are also unable to poison the kegs, says "Ability is not ready yet" when you try to use the poison keg spell or
> "You are too far away" if you right click on it an

Furthermore, and as you can see in this [video](https://www.youtube.com/watch?v=KGDnyHSHHuY):
- The nut toss ability in the marmot's bar will work once and then will keep giving an error message saying it's out of range. I'm unsure about if it's function (as spell's tooltip says) is working.


> Uncompletable atm.

Still uncompletable. You can [interact with the kegs by yourself but nothing happens](https://youtu.be/fcwfttL9FCY). Unsure about if you should be able to click the kegs without being inside the marmot in the first place.

TL:DR Every spell in marmot's bar is bugged somehow or directly not working. You can't complete the quest by yourself either, although you can interact with the kegs directly.

**Comment by fatherfigure2018** [comment]
Hello just 2 add majorbugs in the same chain

1: "The smallest creatures if you click "2" stealth you cant do anything
2: The quest that you travel as a spirit to see in the cave the pike is broken you cant move after you do the spirit thing you have to mount up

ty

**Comment by BezawitNW** [comment]
Quest is still not working

**Comment by bekonja** [comment]
Not working. Also this is quest from a chain means whole chain is failed because of it. Fix asap.

**Comment by naekobest** [comment]
Can confirm, does not work. When stealthed you cant move or do anything at all.

As @bekonja said, it sucks because the whole chain is broken and it is a lot of EXP missing

**Comment by sifiuocz** [comment]
This quest leads to obtaining pre-raid BIS pieces (T4 bis pieces for some classes). It should be fixed ASAP or atleast be autocomplete upon taking the quest.

**Comment by legzor** [comment]
Been trying a few different things to see if there were work arounds.
1: Stealth works if you get far enough away from your character that stands at the Den.
2: Only barrel I could get working was the third barrel ( ...keg of Fermented Seed Beer) Only barrel that you visual can see green poison comming out of it.
3: Tried with 2 people. So one would stand close to barrel while the other person used the Marmot to poisen barrel ( worked on L4G) but no luck.

Hope it can help a little to narrow down what the problem is. (Thought it was something with the Marmor not being registred as your actual character)

**Comment by Evolvee** [comment]
rev 2683

**Comment by Evolvee** [comment]
its not 100% 1:1 retail
but atleast it will be completable for now as a temporary measure
